deferral/ah-django
# #165174845 Show Article Comments

#### What does this PR do?
- Fix comments display on `GET` article requests

#### Description of Task to be completed?
 - [X] Show comments associated with articles

#### How should this be manually tested?
1. Access a clone copy of the repo from [here](https://github.com/andela/ah-django)
```shell
$ git clone https://github.com/andela/ah-django
```
2. Checkout the this bug branch
```shell
$ git checkoutbg-show-article-comments-165174845
```
3. Create a virtual environment and activate it
```shell
$ pip install virtualenv && virtualenv venv && source venv/bin/activate
```
4. Install the project dependencies
```shell
$ pip install -r requirements.txt
```
5. Use the `.env_sample` to create an `.env` file with environment variables then source the .env.
The `env_sample` details out the required environment vairables. Copy them and paste in an `env` file. Export the environment variables with the command:
```
$ source .env
```

6. Send a `POST` request to the endpoint `127.0.0.1:8000/api/articles/` to create
an article. The response body should include the comments posted for that article

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#165174845](https://www.pivotaltracker.com/story/show/165174845)
